reflex/state.py

```diff
@@ -2159,6 +2187,25 @@ async def modify_state(self, token: str) -> AsyncIterator[BaseState]:
             await self.set_state(token, state)
 
 
+# Workaround https://github.com/cloudpipe/cloudpickle/issues/408 for dynamic pydantic classes
+if not isinstance(State.validate.__func__, FunctionType):
+    cython_function_or_method = type(State.validate.__func__)
```

Comment:
so what type is a cython function?

Reply:
a cython function is compiled to C via the cython library. there's no importable name for it though, because it masquerades as a builtin, but it's not accessible as a builtin name.

Reply:
similar to the builtin `function` or `method` types, but those types are exposed via `types.FunctionType` and `types.MethodType` respectively